frontend/src/components/WeekSelection.tsx

```diff
@@ -1,23 +1,22 @@
 "use client";
-import { useFilteredConsultants } from "@/hooks/useFilteredConsultants";
 import SecondaryButton from "@/components/SecondaryButton";
 import { ArrowLeft, ArrowRight } from "react-feather";
 import IconSecondaryButton from "./IconSecondaryButton";
+import { useSelectedWeek } from "@/hooks/staffing/useSelectedWeek";
 
 export default function WeekSelection() {
-  const { incrementSelectedWeek, resetSelectedWeek, decrementSelectedWeek } =
-    useFilteredConsultants();
+  const { changeSelectedWeek, resetSelectedWeek } = useSelectedWeek();
 
   return (
     <div className="flex flex-row gap-1">
       <SecondaryButton label={"Denne uka"} onClick={resetSelectedWeek} />
       <IconSecondaryButton
         icon={<ArrowLeft size={24} />}
-        onClick={decrementSelectedWeek}
+        onClick={() => changeSelectedWeek(-7)}
```

Comment:
As the number of weeks changed is based on number of weeks shown, this needs to use -7, -11 or -23. Same with the function below 😅

Reply:
Good catch! 🎉 This will be changed based on the new `weekSpan` variable coming with the PR from @mathildehaugum :)

Reply:
Nice!